### PR TITLE
add onboarded boolean to users

### PR DESCRIPTION
**Pull request by cwest144**
- default false, switch to true when creating a project for a user.

### CODE REVIEW

File: src/migrations/1674252381160-AddOnboardedToUsers.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+
+export class AddOnboardedToUsers1674252381160 implements MigrationInterface {
+    name = 'sh1674252381160'
```
Comment (jlevers) [inline src/migrations/1674252381160-AddOnboardedToUsers.ts:4]: let's rename this to `AddOnboardedToUsers1674252381160`. in the future, you can control the name by doing something like `yarn migration:generate AddOnboardedToUsers`

File: src/projects/projects.service.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ export class ProjectsService {
   async create(createProjectDto: CreateProjectDto): Promise<Project> {
     const project = await this.projectsRepository.save(createProjectDto);
     const user = await this.usersService.findOne(createProjectDto.userId);
+    this.usersService.update(user.id, {onboarded: true});
```
Comment (jlevers) [inline src/projects/projects.service.ts:20]: this is my bad for not being clearer on slack, but this needs an `await` in front of it

**Review by jlevers — CHANGES_REQUESTED** [review]
couple minor things, but looks good overall :) call him mr. nestjs

once the couple little things are resolved you can merge

File: src/migrations/1674252381160-AddOnboardedToUsers.ts (hunk 1 above)
Comment (cwest144) [reply src/migrations/1674252381160-AddOnboardedToUsers.ts:4]: the filename is currently matching the existing migrations

**Pull request merged**